recipes/angreal/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "angreal" %}
+{% set version = "0.7.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 90582e9014d76e11f14d2f8dbfc4951b9cdc13951948875e65ebd85f232cedcd
+    patches:
+      - 0001-Work-a-round-the-idea-of-parsing-requirements.txt-in.patch
+  - url: https://gitlab.com/angreal/angreal/-/raw/{{ version }}/requirements/requirements.txt
+    sha256: df9fc95dd75893252eea8b481d68dbd3cc09f8b42027f2cb70ce69bcd613ad61
+  - url: https://gitlab.com/angreal/angreal/-/raw/{{ version }}/requirements/dev.txt
+    sha256: 84b4610ee40438b7bf594944b6f148a9390de1473cbc074b71bbd9d23b3607ba
+  - url: https://gitlab.com/angreal/angreal/-/raw/{{ version }}/LICENSE.md
+    sha256: 589ed823e9a84c56feb95ac58e7cf384626b9cbf4fda2a907bc36e103de1bad2
+
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install . -vv
```

Comment:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

Reply:
Thank @marcelotrevisani! I forgot to change it back.

This should be good to go now.